backend/api/Database/Context/FlotillaDbContext.cs

```diff
@@ -70,6 +70,8 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         modelBuilder.Entity<MissionRun>().OwnsOne(m => m.Map).OwnsOne(t => t.TransformationMatrices);
         modelBuilder.Entity<MissionRun>().OwnsOne(m => m.Map).OwnsOne(b => b.Boundary);
+        modelBuilder.Entity<MissionRun>().HasOne(m => m.Area).WithMany();
+        modelBuilder.Entity<Robot>().HasOne(r => r.CurrentArea).WithMany();
```

Comment:
Do we need to define this in the FlotillaDbContext as well as the database models?

Reply:
Same thing as above @mrica-equinor